docker-compose/ups-release/docker-compose-v2.1.yaml

```diff
@@ -2,21 +2,21 @@ version: '2.1'
 
 services:
   unifiedpushserver:
-    image: aerogear/unifiedpush-wildfly:1.2.0
+    image: docker.io/aerogear/unifiedpush-wildfly:1.2.1.1
     volumes:
        - ./helper:/ups-helper
     entrypoint: "/ups-helper/hack.sh"
     depends_on:
        unifiedpushDB:
          condition: service_healthy
     environment:
-        UNIFIEDPUSH_PORT_3306_TCP_ADDR: ${MYSQL_ADDR_UNIFIEDPUSH}
-        UNIFIEDPUSH_ENV_MYSQL_USER: ${MYSQL_USER}
-        UNIFIEDPUSH_ENV_MYSQL_PASSWORD: ${MYSQL_PASSWORD}
-        UNIFIEDPUSH_PORT_3306_TCP_PORT: ${MYSQL_PORT_3306_TCP_PORT}
-        UNIFIEDPUSH_ENV_MYSQL_DATABASE: ${MYSQL_DATABASE_UNIFIEDPUSH}
-        KEYCLOAK_PORT_8080_TCP_ADDR: ${KEYCLOAK_PORT_8080_TCP_ADDR}
-        KEYCLOAK_PORT_8080_TCP_PORT: ${KEYCLOAK_PORT_8080_TCP_PORT}
+        MYSQL_SERVICE_HOST: ${MYSQL_ADDR_UNIFIEDPUSH}
+        MYSQL_USER: ${MYSQL_USER}
+        MYSQL_PASSWORD: ${MYSQL_PASSWORD}
+        MYSQL_SERVICE_PORT: ${MYSQL_PORT_3306_TCP_PORT}
+        MYSQL_DATABASE: ${MYSQL_DATABASE_UNIFIEDPUSH}
+        KEYCLOAK_SERVICE_HOST: ${KEYCLOAK_PORT_8080_TCP_ADDR}
+        KEYCLOAK_SERVICE_PORT: ${KEYCLOAK_PORT_8080_TCP_PORT}
```

Comment:
It might be worth editing the .env file to make the variable names match,

Reply:
ah - right